httproutes.cue

```diff
@@ -0,0 +1,37 @@
+package holos
+
+import v1 "gateway.networking.k8s.io/httproute/v1"
```

Comment:
Note, if we do the following and use it, cue takes ~3 seconds to export instead of 600ms

```cue
import holos "example.com/platform/schemas/holos/v1alpha5"

#HTTPRoutes: holos.#HTTPRoutes & { }
```

I think we may be seeing conflicts over the `holos` field.